sound/soc/sof/intel/byt.c

```diff
@@ -387,13 +387,12 @@ static int byt_get_reply(struct snd_sof_dev *sdev, struct snd_sof_ipc_msg *msg)
 	} else {
 		/* reply correct size ? */
 		if (reply.hdr.size != msg->reply_size) {
-			dev_err(sdev->dev, "error: reply expected 0x%zx got 0x%x bytes\n",
+			dev_err(sdev->dev, "error: reply expected 0x%zx got %u bytes\n",
```

Comment:
Does %u make it obvious to the user that it's base16 ?

Reply:
%u is for size_t and will print in decimal. Andy commented that the sizes should be decimal, not hex and I agree.

Reply:
Oh, I missed that there are two sizes printed in this one, so both have to be made %u. And for size_t it might be %zu, I'll have to double check.